### PR TITLE
Update Jib CLI README.md

### PR DESCRIPTION
**Pull request by mpeddada1**
Rendered: https://github.com/GoogleContainerTools/jib/blob/146a5447d77fbe0d36a6ce618626e5df72be1947/jib-cli/README.md#options

### CODE REVIEW

File: jib-cli/README.md [hunk 1]
```diff
@@ -9,13 +9,29 @@ a Java library for building containers without Docker.
...
+  * [Debugging Params](#debugging-params)
+
 ## Get the Jib CLI
 
 ### Download a java application
 
 A JRE is required to run this Jib CLI distribution.
 
-Find the latest jib-core release on the [Releases page](https://github.com/GoogleContainerTools/jib/releases), download `jib-jre-<version>.zip`, and unzip it. The zip file contains the `jib` (`jib.bat` for Windows) script at `jib/bin/`. Optionally, add the binary directory to your `$PATH` so that you can call `jib` from anywhere.
+Find the latest jib-cli release on the [Releases page](https://github.com/GoogleContainerTools/jib/releases), download `jib-jre-<version>.zip`, and unzip it. The zip file contains the `jib` (`jib.bat` for Windows) script at `jib/bin/`. Optionally, add the binary directory to your `$PATH` so that you can call `jib` from anywhere.
```
Comment (chanseokoh) [inline jib-cli/README.md:34]: Nice catch. Perhaps from now on, for easy access, we should link the latest release and update it whenever we make a new release. For example,
```
Find the [latest jib-cli release 0.2.0](...) on the [Releases page](https://github.com/GoogleContainerTools/jib/releases), download `jib-jre-<version>.zip`, and unzip it. ...
```

File: jib-cli/README.md [hunk 2]
```diff
@@ -30,91 +46,54 @@ $ ./jib-cli/build/install/jib/bin/jib
...
-                              registry
-      --to-username=<username>
-                            username for communicating with target image
-                              registry
-      --username=<username> username for communicating with both target and
-                              base image registries
-      --verbosity=<level>   set logging verbosity, candidates: quiet, error,
-                              warn, lifecycle, info, debug, default: lifecycle
-
+jib build --target ... [options]
```
Comment (chanseokoh) [inline jib-cli/README.md:58]:
```suggestion
jib build --target <image name> [options]
```

File: jib-cli/README.md [hunk 3]
```diff
@@ -30,91 +46,54 @@ $ ./jib-cli/build/install/jib/bin/jib
...
-
-## Build File
-
-The CLI uses a build file to define the container being built. The default is a file named `jib.yaml` in the project root.
-
-### Annotated `jib.yaml`
+### Quickstart
+1. Create a hello world script (`script.sh`) containing:
+    ```
+    echo "Hello World"
```
Comment (chanseokoh) [inline jib-cli/README.md:63]:
```suggestion
    #!/bin/sh
    echo "Hello World"
```

File: jib-cli/README.md [hunk 4]
```diff
@@ -30,91 +46,54 @@ $ ./jib-cli/build/install/jib/bin/jib
...
+2. Create a buildfile. The default is a file named `jib.yaml` in the project root.
+    ```
+    apiVersion: jib/v1alpha1
+    kind: BuildFile
+    
+    from:
+      image: ubuntu
+    
+    entrypoint:
+      - ./script.sh
```
Comment (chanseokoh) [inline jib-cli/README.md:74]:
```suggestion
    entrypoint: ["./script.sh"]
```

(*untested* though)

File: jib-cli/README.md [hunk 5]
```diff
@@ -253,3 +232,95 @@ Parameters that will be overwritten:
 - `workingDirectory`
 - `entrypoint`
 - `cmd`
+
+
+#### Options
```
Comment (chanseokoh) [inline jib-cli/README.md:237]:
```suggestion
### Options
```

File: jib-cli/README.md [hunk 6]
```diff
@@ -253,3 +232,95 @@ Parameters that will be overwritten:
...
+Here is the list of optional flags for the `build` command:
+
+Option | Description
+---     | ---
+`-b, --build-file` |  The path to the build file (ex: path/to/other-jib.yaml)
+`-c, --context`    |  The context root directory of the build (ex: path/to/my/build/things)
+`-p, --parameter`  |  Templating parameter to inject into build file, replace ${<name>} with <value> (repeatable)
+
+
+### Jar
```
Comment (chanseokoh) [inline jib-cli/README.md:247]:
```suggestion
## Jar Command: Containerizing a JAR app
```

File: jib-cli/README.md [hunk 7]
```diff
@@ -253,3 +232,98 @@ Parameters that will be overwritten:
...
+This command follows the following pattern:
+```
+jib jar --target ... path/to/myapp.jar [options]`
```
Comment (chanseokoh) [inline jib-cli/README.md:250]:
```suggestion
jib jar --target <image name> path/to/myapp.jar [options]
```

File: jib-cli/README.md [hunk 8]
```diff
@@ -253,3 +232,98 @@ Parameters that will be overwritten:
...
+```
+
+#### Options
```
Comment (chanseokoh) [inline jib-cli/README.md:253]:
```suggestion
### Options
```

File: jib-cli/README.md [hunk 9]
```diff
@@ -253,3 +232,98 @@ Parameters that will be overwritten:
...
+`--image-format`  | Format of container, candidates: Docker, OCI, default: Docker.
+`--jvm-flags`     | JVM arguments, example: `--jvm-flags=-Dmy.property=value,-Xshare:off`
+`--labels`        | Labels to write into container metadata, example: `--labels=label1=value1,label2=value2`.
+`--mode`          | The jar processing mode, candidates: exploded, packaged, default: exploded
+`--program-args`  | Program arguments for container entrypoint.
+`-u, --user`      | The user to run the container as, example: `--user=myuser:mygroup`.
+`--volumes`       | Directories on container to hold extra volumes, example: `--volumes=/var/log,/var/log2`.
+
+
+### Common Jib CLI Options
```
Comment (chanseokoh) [inline jib-cli/README.md:272]:
```suggestion
## Common Jib CLI Options
```

File: jib-cli/README.md [hunk 10]
```diff
@@ -253,3 +232,98 @@ Parameters that will be overwritten:
...
+The options can either be specified in the command line or defined in a configuration file:
+```
+[@<filename>...]      One or more argument files containing options.
+```
+#### Auth/security
```
Comment (chanseokoh) [inline jib-cli/README.md:277]:
```suggestion
### Auth/Security
```

File: jib-cli/README.md [hunk 11]
```diff
@@ -253,3 +232,98 @@ Parameters that will be overwritten:
...
+```
+    --allow-insecure-registries            Allow jib to send credentials over http (insecure)
+    --send-credentials-over-http           Allow jib to send credentials over http (very insecure)
+```
+
+##### Credentials
```
Comment (chanseokoh) [inline jib-cli/README.md:283]:
```suggestion
#### Credentials
```

**Review by chanseokoh — APPROVED** [review]
Let's merge this early. We can iterate over this.

File: jib-cli/README.md (hunk 3 above)
Comment (mpeddada1) [reply jib-cli/README.md:63]: Good catch.

File: jib-cli/README.md (hunk 1 above)
Comment (mpeddada1) [reply jib-cli/README.md]: Good idea.

**Pull request merged**